packages/string-templates/scripts/gen-collection-info.ts

```diff
@@ -34,15 +33,13 @@ const outputJSON: Manifest = {}
 const ADDED_HELPERS = {
   date: {
     date: {
-      args: ["datetime", "format"],
-      numArgs: 2,
+      args: ["[datetime]", "[format]", "[options]"],
```

Comment:
Why are the square brackets necessary here?

Reply:
This is the [JSDoc](https://jsdoc.app/tags-param) code for optional parameters